ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,175 @@
+
+A workflow runs from left to right, from a start point to an end point, which can be in turn linked to another workflow. A workflow must contain at least one action between the start and the end point, which are special types of places. The actions are linked through places with each other, connecting the corresponding actions input and output ports. All these elements are described more in detail below.
+
+> [Info] The *Workflows* module is linked to different system modules and plugins, such as *Invoicing*, *Email*, *OmniChannels*, *Tasks*, among others. The actions available to create workflows depend on the modules installed in the system.
+
+[comment]: <> (Processes - The data input at the start point, for example a document, goes through the different workflow stages -places and actions- until it reaches the end point. The process is than completed.)
+
+
+
+## Places
+
+![Place](/Assets/Screenshots/ActindoWorkFlow/Workflows/Place.png "[Place]")
+
+A place is a step in the workflow and is represented by circle, usually provided with inward and outward arcs. Places serve as link between two actions, joining an output port from the previous action to the corresponding input port of the following action. Places linking mandatory input/output ports are displayed by default and cannot be deleted. Additional places can be added by clicking on the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button next to the optional input/output port. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-ports).
+
+When clicking a place, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar01.png "[Settings side bar]")
+
+The place settings side bar is located to the right of the workspace and allows to define the place settings. The following fields are displayed:
+
+  - *Key*  
+  Place key. This field is read-only. The key is required for API access and must be system wide unique.
+
+  - *Data container*  
+  Use this field to define the data type to be used in the place. If necessary, click the ![Delete](/Assets/Icons/Cross02.png "[Delete]") (Delete) button to display the whole list of available data types. Enter a search text for the desired data type. The list of actions is filtered for your search text as you type. Select the ![Radio button](/Assets/Icons/Radiobutton.png "[Radio button]") (Radio button) corresponding to the appropriate option.
+
+
+
+## Actions
+
+![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/AddAction.png "[Action]")
+
+Actions are the individual activities being executed in the workflow and are represented by a rectangle.
```

Comment:
Add info from gitlab:
Every API endpoint of the modules automatically becomes available as an action and can be used in any workflow - but you can also build your own actions independent from the api.